dozer-types/protos/api_explorer.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package dozer.api_explorer;
+service ApiExplorerService {
+    rpc GetApiToken(GetApiTokenRequest) returns (GetApiTokenResponse);
+}
+
+message GetApiTokenRequest {
+  // required if in cloud context.
+  optional string app_id = 1;
```

Comment:
right now, in Cloud context, the deployment number is not really helpful in this case, since we have Variable per application, not per deployment. So app_id is enough for fetching secret to generate token. I think we have to revise this with the case multiple APIs is running at same time. Should secret token is different per API ?

Reply:
Hmm, I guess user has to live with the fact that api secret cannot be changed.

Reply:
I have one more question, what would happen if the user uses `{{}}` syntax and pass api secret as a secret?

Reply:
right, i still not consider this case. It gonna conflict with the rule we have previously: Prior Env over config
For instance, we input `!JWT {{VARIABLE_SECRET}} `
=> sending to cloud and deploy => env DOZER_MASTER_SECRECT = {{VARIABLE_SECRET}}
=> when the `dozer` binaray is called inside pod, `!JWT {{VARIABLE_SECRET}} ` will be binding with actuall value => so the Api Config will have the correct value, not the DOZER_MASTER_SECRECT

=> Maybe for this case, on `dozer` repo logic, we should prior the config over variable ? What do you think about this @chubei

Reply:
Agreed, prior the config over variable will solve this problem.

Another problem remains, the `DOZER_MASTER_SECRECT` being the literal `{{VARIABLE_SECRET}}` will prevent the `ApiExplorerService` from working.

Reply:
How about giving instructions to the user for generating token and guide them to put into the API explorer?

Reply:
> How about giving instructions to the user for generating token and guide them to put into the API explorer?

hmm, that's not a really convenient experience on API Explore
I was thinking a way like this: 
on `dozer` binary, when running, we add extra step logic =>  if api config jwt having value, we set env DOZER_MASTER_SECRECT = jwt secret value
=> it should work right ?, so next time we query DOZER_MASTER_SECRECT from ApiExplorerService, correct value should be there

Reply:
If we're going to make the secret plain text, we can just read it from configuration and save it to k8s, without going through environment variable. And I'm not sure if we can let a container pass information to k8s through environment variables.

Reply:
oh, then a more simple approach, everything stay the same
we have a list of secrets already
just try bind DOZER_MASTER_SECRET with `handlebars` before send it to `deployer` along with secrets list

Reply:
yup

Reply:
include this logic on cloud [PR](https://github.com/getdozer/dozer-cloud/pull/554/files)
can you check and approve it also @chubei  ? these PRs should go together